### DISCUSSION TITLE
Re: Added more statistics such as GPU, CPU, Version and Algorithm graphs

### DISCUSSION THREAD

**Discussion by Seliarini** [UnamSanctam/UnamWebPanel#256] [Q&A]
Hey Unam, love the new update. Noticed you this in the changelog: added more statistics such as GPU, CPU, Version and Algorithm graphs.

My question is how were you able to detect GPU and CPU versions without triggering AV? Cos I know getting computer's credentials often lead to increase risk in getting detected by dynamic analysis malware confidence level score

**Comment by UnamSanctam** [comment]
Currently it uses the WMI to get the GPU and CPU names, while using the WMI can be flagged by the antivirus in this case it doesn't really change much since I have to use the WMI for other things anyways at the moment (haven't found any other ways to retrieve some data, such as my custom idle detections that also works when running as "System"). You can get it using alternative means, such as how the miner installer/injector checks for a GPU https://github.com/UnamSanctam/SilentCryptoMiner/blob/8caef3eec1040ddb982574ccd7cc3d88016f7c09/SilentCryptoMiner/Resources/Code/Includes/common.cpp#L141.
